OAuth: Allow assigning Server Admin (#54780)
grafana/grafana: docs/sources/setup-grafana/configure-security/configure-authentication/azuread.md

> 1. Click on **Users and Groups** and add Users/Groups to the Grafana roles by using **Add User**.

Reviewer: This step confuses me a bit. Is the user clicking **Users and Groups** and then clicking **Add User**? I'm not sure what you mean by 'using Add User'.

Author: this is not related to this PR and no change was applied here
.
I've checked Azure's UI and this is indeed what the author of the line meant: The user clicks Users and Groups and then clicks Add User once they arrive on the new menu

### Assign server administrator privileges

> Available in Grafana v9.2 and later versions.

> If the application role received by Grafana is `GrafanaAdmin`, the user will be granted server administrator privileges.  

Reviewer: ```suggestion
If the application role received by Grafana is `GrafanaAdmin`, Grafana grants the user server administrator privileges.  
```

This is useful if you want to grant server administrator privileges to a subset of users.  

> The user will also be assigned the role of `Admin` of the default organization.

Reviewer: This is passive voice. Who or what is doing the assigning? In my suggestion, I am making an assumption that the 'actor' is Grafana.

> The user will also be assigned the role of `Admin` of the default organization.

Reviewer: ```suggestion
Grafana also assigns the user the `Admin` role of the default organization.
```

The setting `allow_assign_grafana_admin` under `[auth.azuread]` must be set to `true` for this to work.  

> If the setting is set to `false`, the user will be assigned the role of `Admin` of the default organization, but not server administrator privileges.

Reviewer: ```suggestion
If the setting is set to `false`, the user is assigned the role of `Admin` of the default organization, but not server administrator privileges.
```
